Makefile

```diff
@@ -119,7 +119,7 @@ help: ## Display this help.
 
 .PHONY: manifests
 manifests: controller-gen kustomize ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="{./apis/...,./cmd/...,./controllers/...,./internal/...,./sidecar/...}" output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="{./api/...,./cmd/...,./internal/controller/...,./internal/...,./sidecar/...}" output:crd:artifacts:config=config/crd/bases
```

Comment:
Is it required to move the public API from `apis` to `api`? Other projects that use this API will break with this change. If there is a good reason to do so, it needs to be explained clearly in the commit message. Maybe place this renaming of the directory in a separate commit so that it stands out a little more.

Reply:
Unfortunately, yes. I will update the commit message, along with moving the renames under api to its own commit.